### ISSUE TITLE
Lag unit is based on viewPreset unit when setting lag for predecessors/successors

### ISSUE THREAD

**Issue by matsbryntse** [bryntum/support#2728]
[Forum post](https://www.bryntum.com/forum/viewtopic.php?f=52&t=17294&p=85806#p85806)

Hi there

I have a question regarding adding new predecessors/successors with lags using the left side of gantt chart. When we want to add a relation between tasks, for example I want to set the predecessor to task 2, with lag of 3 days, if I omit the unit and just press enter, the unit is automatically calculated based on the zoom level of the right hand side of the gantt chart.
If the zoom is on quarter levels, by typing 
```
2+3Q
```, and if we are zoomed in, we get 
```
2+3D
```.

Here is the config code for the column:


```
      {
        id: 'predecessor',
        type: 'predecessor',
        filterable: false,
        editor: {
          picker: {
            width: 256,
          },
          placeholder: `${t('search')}...`,
          icon: 'b-icon-search',
        },
        dependencyIdField: 'taskNumber',
        width: 112,
      },

```

We want to make this consistent and only use day units and not be dependant on zoom level.
How can we do that?
![Before hitting enter](https://www.bryntum.com/forum/download/file.php?id=13070)
![After hitting enter on zoomed out level](https://www.bryntum.com/forum/download/file.php?id=13069)

**Comment by vahid-capmo** [comment]
Hi @SergeyMaltsev 
Did this fix actually land on 4.1.1? Because I don't see it in the change log of that release.

**Comment by SergeyMaltsev** [comment]
Yes it is in 4.1.1
You may check it here
https://www.bryntum.com/products/gantt/changelog/
